services/ingest.json

```diff
@@ -17,6 +17,24 @@
         "signature": "4faefcc46299557efaa1c896835a2413ad0646201e40938b4207dbd4cfecf33303cf5f28fd1f53cb71146c45d12c7fa34d2d87d312acf9fd7964c08506d2278d"
       }
     },
+    {
+      "groupId": "io.jenkins.plugins",
+      "artifactId": "aws-global-configuration",
+      "url": "https://repo.jenkins-ci.org/incrementals/io/jenkins/plugins/aws-global-configuration/1.0-rc20.ffbe7e5de068/aws-global-configuration-1.0-rc20.ffbe7e5de068.hpi",
+      "checksum": {
+        "type": "sha512",
+        "signature": "c78067f7f96668aa9798b7fc735b77dfaa208f4499238d42079ba6a4c9eabbc43adf6b58b9e5cb1dd143b17659f33d23945cda86d09423e17c4b9dad682c5cef"
+      }
+    },
+    {
+      "groupId": "io.jenkins.plugins",
+      "artifactId": "artifact-manager-s3",
```

Comment:
Apparently `ingest-update-center` neglects to sort the plugin list, so diffs will be hard to verify.